File: Web/src/main/java/com/itechart/webflux/web/core/repository/MeetingHistoryRepository.java
package com.itechart.webflux.web.core.repository;

import com.itechart.webflux.web.core.model.entity.MeetingHistory;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;

public interface MeetingHistoryRepository extends ReactiveMongoRepository<MeetingHistory, String> {



}

File: Web/src/main/java/com/itechart/webflux/web/core/repository/MeetingHistoryRepository.kt
package com.itechart.webflux.web.core.repository

import com.itechart.webflux.web.core.model.entity.MeetingHistory
import org.springframework.data.mongodb.repository.ReactiveMongoRepository

interface MeetingHistoryRepository : ReactiveMongoRepository<MeetingHistory, String>